### ISSUE TITLE
508-defect-2 [COGNITION, SCREENREADER]: There's a typo in the screen-reader only text, pre-pending with a $

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12190]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

I believe there is a typo in the screen reader only text, pre-pending each text with a dollar sign.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user I want to clearly understand the content of the links.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Log in as user 1, enter https://staging.va.gov/manage-va-debt/your-debt/ in browser
2. Navigate to the debt letters section
3. Inspect the code
4. Verify  the dollar sign appears, such as `<span class="sr-only">` $Second Demand Letter `</span>`

### Current code (address typo)

```diff
<a 
      download="Second Demand Letter dated May 29, 2020" 
      href="https://staging-api.va.gov/v0/debt_letters/%7B64B0BDC4-D40C-4C54-86E0-104C987B8D8F%7D">
         <i aria-hidden="true" role="img" class="fas fa-download vads-u-padding-right--1"></i>
         Download
-         <span class="sr-only">` $Second Demand Letter `</span> 
         letter 
         <span class="sr-only">`dated $May 29, 2020`</span> 
         <dfn><abbr title="Portable Document Format">(PDF)</abbr></dfn>
</a>
```

### Proposed code

```diff
<a 
      download="Second Demand Letter dated May 29, 2020" 
      href="https://staging-api.va.gov/v0/debt_letters/%7B64B0BDC4-D40C-4C54-86E0-104C987B8D8F%7D">
         <i aria-hidden="true" role="img" class="fas fa-download vads-u-padding-right--1"></i>
         Download
+         <span class="sr-only">` Second Demand Letter `</span> 
         letter 
         <span class="sr-only">`dated $May 29, 2020`</span> 
         <dfn><abbr title="Portable Document Format">(PDF)</abbr></dfn>
</a>
```

![Screen Shot 2020-08-12 at 11 09 08 AM](https://user-images.githubusercontent.com/57469/90032364-4dfa4180-dc8c-11ea-9e9e-07fc42fbac17.png)

**Comment by bradl3yC** [comment]
@jenstrickland this has been resolved and is ready to be validated in staging

**Comment by jenstrickland** [comment]
I'm sorry, @bradl3yC — I'm still seeing this, even after a hard reload. Is it possibly due to it happening in a couple of the sr-only spans?

![Screen Shot 2020-08-12 at 2 09 47 PM](https://user-images.githubusercontent.com/57469/90051421-aa1d8f80-dca5-11ea-9d65-b2fd858e05d3.png)

**Comment by bradl3yC** [comment]
@jenstrickland should be resolved now - please verify

**Comment by jenstrickland** [comment]
Validated. Thank you!
